examples/jobs/.saturn/saturn.json

```
@@ -0,0 +1,13 @@
+{
+    "image": "saturncloud/saturn:2021.09.20",
+    "deployment": {
+        "size": "large",
+        "command": "papermill example.ipynb out.ipynb --stdout-file - --stderr-file - --kernel python3"
+    },
+    "environment_variables": {},
+    "description": "Run a job on Saturn Cloud. Read more about jobs at: https://saturncloud.io/docs/using-saturn-cloud/jobs_and_deployments/ .",
```

Nitpick, but can we make a short link for this like we do in the dashboard example?

---

Specifically that would be

```suggestion
    "description": "Run a job on Saturn Cloud. Read more about jobs at: https://scld.io/docs/deployments .",
```